### ISSUE TITLE
 .vscode/settings.json should maybe not be under source control

### ISSUE THREAD

**Issue by lbovet** [enola-dev/enola#545]
## Expected Behavior

No changed files after initial build

## Actual Behavior

 .vscode/settings.json has changes, maybe IDE-generated files should not be under source control

- Version: 2ce7768f42d2aee2cee8c9cfe82db96deca99387
- Platform: Ubuntu 20.04

**Comment by vorburger** [comment]
> maybe IDE-generated files should not be under source control

May I _disagree_ with this? Enola's [`.vscode/settings.json`](https://github.com/enola-dev/enola/blob/main/.vscode/settings.json) contains a number of settings which are required to work on this project with VSC. There is no "other source" for such settings (that I know of) which "duplicates" these, or from which they could be "generated". As such, my view would be that this file does belong under source control. Would you still disagree, given this additional new information for context?

> No changed files after initial build Actual Behavior .vscode/settings.json has changes,

Ah, but that is **do agree** with! 😄  Let's perhaps re-focus this issue on seeing if we can do anything to improve that - OK for you?

What is "dirty", what's the "diff" that you are seeing, after an _"initial build"?_ (Are you running `./test.bash` for that?)

PS: You technically ARE free [to use another](https://docs.enola.dev/dev/ide/) IDE than VSC, if you prefer!

**Comment by lbovet** [comment]
Yes, the issue is the dirty state after build, not having the file under source control.

Here is the diff:
```
index 128cd16..90fb0a6 100644
--- a/.vscode/settings.json
+++ b/.vscode/settings.json
@@ -128,10 +128,16 @@
     "**/CVS": true,
     "**/.DS_Store": true,
     "**/Thumbs.db": true,
+    ".gitignore": true,
+    ".pytest_cache": true,
+    ".vscode/": true,
+    "**/__pycache__/": true,
+    "**/*.pyc": true,
+    "**/obj/": true,
     "**/.classpath": true,
-    "**/.factorypath": true,
     "**/.project": true,
     "**/.settings": true,
+    "**/.factorypath": true,
     "**/*.crswap": true,
     "**/eclipse-bin/": true,
     "**/eclipse-testbin/": true,

**Comment by vorburger** [comment]
> -    "**/.factorypath": true,
> +    "**/.factorypath": true,

This ^^^ I've also noticed keeps "jumping around" and changing position - and I've no idea why! I think it may be different between different versions of... well, I'm not entirely sure what! Is it VSC itself that changes this? Or is one of the (many) code formatters that I have set up in this project perhaps doing this? E.g. slightly different local versions of... `prettier` - maybe? I would also like to fix that, and welcome investigations into how!

> +    ".gitignore": true,
> +    ".vscode/": true,

This is very curious, because it's `".vscode": false,` [here](https://github.com/enola-dev/enola/blob/2ce7768f42d2aee2cee8c9cfe82db96deca99387/.vscode/settings.json#L148)... which VSC version do you use? I have 1.86.2..

> +    ".pytest_cache": true,
> +    "**/__pycache__/": true,
> +    "**/*.pyc": true,
> +    "**/obj/": true,

These I have never seen! You probably have a more fancy Python environment than me? Or some Python related VSC extensions? Would you like to raise a PR to add at least these 4 already? I'm happy to merge that ASAP! (Maybe let's already do just these 4, to start with; I think if you raise a PR for the other ones they will just keep getting changed again by future changes from me.)
